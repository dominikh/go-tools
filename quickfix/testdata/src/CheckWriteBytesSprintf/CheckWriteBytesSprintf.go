package pkg

import (
	"fmt"
	"io"
)

func _(w io.Writer) {
	w.Write([]byte(fmt.Sprint("abc", "de")))   /* want `should use fmt.Fprint` */
	w.Write([]byte(fmt.Sprintf("%T", w)))      /* want `should use fmt.Fprintf` */
	w.Write([]byte(fmt.Sprintln("abc", "de"))) /* want `should use fmt.Fprintln` */
}
