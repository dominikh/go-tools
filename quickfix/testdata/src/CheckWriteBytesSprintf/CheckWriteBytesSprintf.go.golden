package pkg

import (
	"fmt"
	"io"
)

func _(w io.Writer) {
	fmt.Fprint(w, "abc", "de")   /* want `should use fmt.Fprint` */
	fmt.Fprintf(w, "%T", w)      /* want `should use fmt.Fprintf` */
	fmt.Fprintln(w, "abc", "de") /* want `should use fmt.Fprintln` */
}
